bin/check

```diff
@@ -22,12 +22,20 @@ end.freeze
 # If there are no options given, invoke help
 ARGV << "-h" if ARGV.empty?
 
+BLURBS = ALL_CHECKS.map do |ch|
```

Comment:
`BLURBS` could maybe be something like `CHECK_DESCRIPTIONS` or `DESCRIPTIONS`? I think for the `map` it reads fine, just in the `Available checks:` section `#{BLURBS}` seems _subjectively_ a bit more questionable.

Reply:
fiiiine